projects/plugins/protect/src/class-jetpack-protect.php

```diff
@@ -454,4 +455,13 @@ public static function get_waf_stats() {
 			'automaticRulesLastUpdated' => Waf_Stats::get_automatic_rules_last_updated(),
 		);
 	}
+
+	/**
+	 * Get WAF standalone mode status
+	 *
+	 * @return bool|array True if WAF standalone mode is enabled, false otherwise.
+	 */
+	public static function get_waf_standalone_mode_status() {
+		return defined( 'JETPACK_WAF_RUN' ) && JETPACK_WAF_RUN === 'preload';
+	}
```

Comment:
This might fit nicely in the `waf` package, similar to how we are using `Waf_Runner::get_config()` and `Waf_Stats::get_automatic_rules_last_updated()` to map data from the firewall into the plugin's default state.

Reply:
Agreed! Made the change 👍🏻